### PR TITLE
New package: tickr

### PR DESCRIPTION
**Pull request by benalb**

### CODE REVIEW

File: srcpkgs/tickr/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'tickr'
+pkgname=tickr
+version=0.6.4
+revision=1
+build_style=gnu-configure
+hostmakedepends="gtk+-devel libxml2-devel"
+makedepends="pkg-config"
```
Comment (Johnnynator) [inline srcpkgs/tickr/template]: You mixed hostmakedepends and makedepends up. It should be the other way around.
```
hostmakedepends="pkg-config"
makedepends="gtk+-devel libxml2-devel"
```

File: srcpkgs/tickr/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+short_desc="A gtk-based rss reader in a ticker bar"
```
Comment (cr6git) [inline srcpkgs/tickr/template]: `short_desc="GTK-based ...`

File: srcpkgs/tickr/template [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+maintainer="Benjamín Albiñana <benalb@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://www.open-tickr.net/"
+distfiles="https://www.open-tickr.net/src/${pkgname}-${version}.tar.gz"
+checksum=8476fa8f539487c37617855b8312dbfaec322fe51fa5d9e9abe8c093caad75bd
+
+post_install() {
+	vlicense COPYING
```
Comment (cr6git) [inline srcpkgs/tickr/template]: no need to add the license.

File: srcpkgs/tickr/template (hunk 1 above)
Comment (benalb) [reply srcpkgs/tickr/template]: will fix

File: srcpkgs/tickr/template (hunk 3 above)
Comment (benalb) [reply srcpkgs/tickr/template]: will fix both.

File: srcpkgs/tickr/template [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'tickr'
+pkgname=tickr
+version=0.6.4
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="gtk+-devel libxml2-devel"
```
[the remark this answers is not in the document]
Comment (benalb) [reply srcpkgs/tickr/template]: the configure script search for gtk+-2.0 dev lib.

**Pull request merged**